lib/order.rb

```diff
@@ -0,0 +1,36 @@
+module Order
+=begin
+def order_input(menu)
+  x = 1
+  while x == 1
+    y = 1
+    while y == 1
+      puts "Enter the name of the dish. (Enter q to quit)"
+      dish = gets.chomp.to_sym
+      fail "User has cancelled order" if dish == :q
+      if menu.has_key?(dish)
+        y = 2
+        dish_array << dish
+      else
+        "Sorry, did not recognise that dish."
+      end
+    end
+    while z == 1
+      puts "Enter the quantity you require. (Enter q to quit)"
+      quantity = gets.chomp
+      fail "User has cancelled order" if quantity == "q"
+      puts "You entered #{quantity.to_i}, is that correct? Enter y to continue with your order"
```

Comment:
Metrics/LineLength: Line is too long. [95/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)